server/lib/validation/form_validators/ManageApplication.js

```diff
@@ -36,8 +36,10 @@ class ManageApplication extends Validator {
           this.errors.description = errorManifest.description.invalid;
         }
       }
-      if (typeof payload.environment === 'undefined' || payload.environment === '') {
-        this.errors.environment = errorManifest.environment.blank;
+      if (isUpdate === false) {
+        if (typeof payload.environment === 'undefined' || payload.environment === '') {
+          this.errors.environment = errorManifest.environment.blank;
+        }
```

Comment:
Would it make more sense to set up different validation methods for post and update?

Reply:
Refactored validate method out to addApplication & updateApplication